Slim/Plugin/Podcast/Plugin.pm

```diff
@@ -312,11 +329,15 @@ sub searchHandler {
 	)->get($url, @$headers);
 }
 
-sub _precacheShowDetails {
+sub precacheFeedData {
 	my ($url, $feed) = @_;
```

Comment:
Please add some sanity checking here. You're feeding this function whatever you get back from the parser. Make sure `$url` is defined, and `$feed` is more or less what we would expect.

Reply:
Point taken.
I'll append another commit to take up the issue. If looks ok, then can be squashed into existing commits as appropriate.